vaas/vaas/router/admin.py

```diff
@@ -1,11 +1,22 @@
 from django.contrib import admin
 
 from vaas.external.audit import AuditableModelAdmin
-from vaas.router.models import Route
+from vaas.router.models import Route, Rewrite, RewritePositiveUrl
 from vaas.router.forms import RouteModelForm
 from django.conf import settings
 
 
+class RewritePositiveUrl(admin.TabularInline):
+    model = RewritePositiveUrl
+    extra = 1
```

Comment:
what is the purpose of this field?

Reply:
Only reason for do that was to exposed it in admin form. To show connection between Rewrites and RewritePositiveUrl. So it is temporarily. Should disapired after form will be add.